python/ray/job_config.py

```diff
@@ -1,6 +1,16 @@
 import ray
 
 
+class RuntimeEnv:
+    def __init__(self,
+                 working_dir: str = None,
+                 working_dir_uri: str = None,
+                 local_modules=[]):
```

Comment:
How about we omit local_modules in this initial PR?

Reply:
Sure, but any reason for this? I mean it's working, right?

Reply:
It will make it easier to review. When adding a new critical feature, we should try to keep the initial PR very minimal since it's introducing a lot of new interfaces.

Later on, it's quite easy to add features once the initial framework is there.

Reply:
Ok. Do you think it's ok for it to be here? I mean it's just a small feature and not too many lines of code. I've tested it and implemented it.

I'll put other things in another PR.